packages/medusa-js/src/index.ts

```diff
@@ -1,6 +1,6 @@
 import MedusaError from "./error"
 import KeyManager from "./key-manager"
-import Client, { Config } from "./request"
+import Client, { type Config } from "./request"
```

Comment:
suggestion: Can't you just alias the import and revert the export?

Reply:
I don't see how this resolves the issue? We can't do `export { Config } from "/request"`

Reply:
<img width="1116" alt="CleanShot 2023-07-11 at 13 00 09@2x" src="https://github.com/medusajs/medusa/assets/59018053/c0c99348-6e94-4d63-8e0f-e2fafaa1f20e">

Reply:
Just to be sure, what you want is:
```
import { Config as SomeAlias } from "./request"
// and
export { Config } from "./request"
```

right?

Reply:
As discussed, I found out that tsup will use SWC in case the metadata flags are enabled in the tsconfig. In that case, swc removed the exported types which means that if a file is treated after then the export type from will throw since the type will not exist anymore. this is a limitation of the bundler and therefore we will remove those flags since we are not using them